fs/operations/operations.go

```diff
@@ -1399,7 +1400,12 @@ func RcatSize(fdst fs.Fs, dstFileName string, in io.ReadCloser, size int64, modT
 
 // CopyURL copies the data from the url to (fdst, dstFileName)
 func CopyURL(fdst fs.Fs, dstFileName string, url string) (dst fs.Object, err error) {
-	resp, err := http.Get(url)
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: fs.Config.InsecureSkipVerify},
+	}
+	client := &http.Client{Transport: tr}
```

Comment:
I think using the rclone client might be best here

```go
    client := fshttp.NewClient(fs.Config)
```

That will make --no-check-certificate work.  It will also make a host of other things work like `--timeout`, `--dump headers`, etc so it is probably the best solution